mscore/palette.cpp

```diff
@@ -786,6 +787,10 @@ void PaletteScrollArea::keyPressEvent(QKeyEvent* event)
                   p->update();
                   break;
                   }
+            case Qt::Key_Enter:
+            case Qt::Key_Return:
+                  p->applyPaletteElement();
```

Comment:
This change causes New Score Wizard also apply key signature to the currently opened score on pressing Enter. Probably Enter/Return key behavior should honor the value of [`_disableDoubleClick`](https://github.com/musescore/MuseScore/blob/d42cab1e89cf700fcefddeb895f24a358d3e5bf4/mscore/palette.cpp#L806) flag (which got renamed to `_disableElementsApply` in #5546)

Reply:
Good catch!  I remember the same bug being fixed for double-click years ago, I guess that's when the flag got introduced.

I will update this to use _disableDoubleClick, on the assumption this PR will likely be merged before the other.

Reply:
OK, guess I was wrong about the other PR not being merged so quickly :-).  Was about to submit PR, but now will rebase & rebuild.